src/components/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { CaretSortIcon } from '@radix-ui/react-icons';
+import { Popover, PopoverContent, PopoverTrigger } from '../Popover';
+import { Button } from '../Button';
+import { Command } from '../Command';
+import {
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from '../Command/Command';
+import { cn } from '../../lib/utils';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+
+const frameworks = [
```

Comment:
Die haben in der Combobox nix verloren.
Schließlich haben die nichts mit der eigentlichen funktion zu tun